lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,16 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
+#[10110010]
+#[76543210]
+require 'pry'
+
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  decimal = 0
+  binary_array.each_with_index { |number, index|
+    if number == 1
+      decimal += 2 ** (7 - index)
```

Comment:
This definitely works, but I suggest you either make 7 a constant, or use `binary_array.length - 1` instead.